src/components/cards/menuCard/PEMenuCard.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import PEMenuCard from './PEMenuCard';
+
+const meta: Meta<typeof PEMenuCard> = {
+    title: 'Cards/PEMenuCard',
+    component: PEMenuCard,
+};
+
+export default meta;
+
+export const Component: StoryObj<typeof PEMenuCard> = {
+    args: {
+        pricePerPerson: 12,
+        kitchens: ['PEMenuCard', 'PEMealC', 'PEMenuCard'],
+        pictures: [
+            'https://prykoly.ru/wp-content/uploads/2019/11/Chef9.jpg',
+            'https://prykoly.ru/wp-content/uploads/2019/11/Chef9.jpg',
+            'https://prykoly.ru/wp-content/uploads/2019/11/Chef9.jpg',
+        ],
+        chefPicture: 'https://prykoly.ru/wp-content/uploads/2019/11/Chef9.jpg',
+        categories: ['PEMenuCard', 'PEMealC', 'PEMenuCard'],
+        description: 'PEMenuCard PEMenuCard PEMenuCard PEMenuCard PEMenuCard PEMenuCard PEMenuCard PEMenuCard',
+        title: 'PEMenuCard PEMenuCard PEMenuCard PEMenuCard PEMenuCard PEMea',
+        chefName: 'Loren',
```

Comment:
I would suggest to use the property names as mock data, so we know what we are looking at in the storybook.

```suggestion
        categories: ['category-1', 'category-2', 'category-3'],
        description: 'Menu descriptio',
        title: 'Menu title,
        chefName: 'Loren',
```